### DISCUSSION TITLE
How to restore the old quick settings layout?

### DISCUSSION THREAD

**Discussion by Matihood1** [lllyasviel/stable-diffusion-webui-forge#1195] [Q&A]
So the recent update (or one of the more recent updates) completely changed the quick settings bar, introducing the new preset radio buttons and throwing my existing custom quick settings out of the window. Needless to say, I'm not a fan. Also, the checkpoint picker is now too narrow and I get a horizontal scrollbar when I want to pick a model. How do I restore the old layout?
![image](https://github.com/user-attachments/assets/aa2b4471-a159-422d-bc73-8bd553773378)

**Comment by lllyasviel** [comment]
hi you can use https://github.com/AUTOMATIC1111/stable-diffusion-webui or https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions/849 to have old layouts

**Reply by Matihood1** [reply]
Switching to the regular Web UI is a no for me because I can't use SDXL without Forge (I only have 3060). I kinda figured I had to grab an older version. A shame...

**Reply by shaun-ba** [reply]
.
